component/navbar.tsx

```diff
@@ -0,0 +1,38 @@
+import styles from "../styles/Navbar.module.css";
+import type { NextPage } from 'next'
+export const Navbar: NextPage = ({children})=>{
```

Comment:
lets make this a type `React.FC()`. Take a look at the components on main for an example